src/main/java/org/cactoos/text/RandomText.java

```diff
@@ -0,0 +1,124 @@
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import java.io.IOException;
+import java.security.SecureRandom;
+import java.util.List;
+import java.util.Random;
+import org.cactoos.Text;
+import org.cactoos.list.ListOf;
+
+/**
+ * Random text.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Roman Proshin (roman@proshin.org)
+ * @version $Id$
+ * @since 0.32
+ */
+public final class RandomText implements Text {
+
+    /**
+     * Max length of generated text (if no length is specified).
+     */
+    private static final int MAX_RANDOM_LENGTH = 255;
+
+    /**
+     * List of characters allowed for generating.
+     */
+    private final List<Character> characters;
+
+    /**
+     * Length of generated text.
+     */
+    private final int length;
+
+    /**
+     * Characters index randomizer.
+     */
+    private final Random random;
+
+    /**
+     * Ctor.
+     */
+    public RandomText() {
+        this(new SecureRandom().nextInt(RandomText.MAX_RANDOM_LENGTH - 1) + 1);
+    }
+
+    /**
+     * Ctor.
+     * @param len Length of generated text.
+     */
+    public RandomText(final int len) {
+        this(
+            new ListOf<>(
+                '!', '"', '#', '$', '%', '&', '\'', '(', ')', '*',
+                '+', ',', '-', '.', '/', '0', '1', '2', '3', '4',
+                '5', '6', '7', '8', '9', ':', ';', '<', '=', '>',
+                '?', '@', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H',
+                'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R',
+                'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '[', '\\',
+                ']', '^', '_', '`', 'a', 'b', 'c', 'd', 'e', 'f',
+                'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p',
+                'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
+                '{', '|', '}', '~'
+            ),
+            len
+        );
+    }
+
+    /**
+     * Ctor.
+     * @param chrs List of characters allowed for generating.
+     */
+    public RandomText(final List<Character> chrs) {
```

Comment:
@proshin-roman Add one more ctor `RandomText(final Character... chrs)` to simplify usage.